### PR TITLE
Migrate travis to github actions

### PR DESCRIPTION
**Pull request by ecolautti-form3**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        with:
+          fetch-depth: 0
+      - name: Setup Go
+        uses: actions/setup-go@v3.5.0
+        with:
+          go-version-file: 'go.mod'
+      - name: Run tests
+        id: runTests
+        run: make test
+        env:
```
Comment (elliotwms-f3) [inline .github/workflows/ci.yaml:22]: Looks like one of the tests are being skipped due to missing env vars:

https://github.com/form3tech-oss/go-form3/actions/runs/4185577289/jobs/7252823635#step:4:14

**Review by elliotwms-f3 — COMMENTED** [review]
LGTM other than the comment. Does this close https://github.com/form3tech-oss/go-form3/pull/75?

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (ecolautti-form3) [reply .github/workflows/ci.yaml:22]: Added missing env vars, should be fine now

Comment (ecolautti-form3) [reply .github/workflows/ci.yaml:22]: > LGTM other than the comment. Does this close #75?

yes, I'll close it

**Review by elliotwms-f3 — APPROVED** [review]

**Pull request merged**